### PR TITLE
Test storage location

### PR DESCRIPTION
**Pull request by fiiiu**
Added a test that exposes how storage is taking place in the KernelProxy contract.

### CODE REVIEW

File: kernel/test/KernelProxy.test.js [hunk 1]
```diff
@@ -38,9 +38,9 @@ contract('zeppelin_os', ([_, zeppelin, developer, someone, anotherone]) => {
 
     // deploy another instance of the testing contract
     const { logs: logs2 }  = await factory.createProxy(distribution, version, contractName);
-    const proxyAddress2 = logs2.find(l => l.event === 'ProxyCreated').args.proxy;
-    this.mock2 = await PickACard.new(proxyAddress2);
-
+    this.proxyAddress2 = logs2.find(l => l.event === 'ProxyCreated').args.proxy;
+    this.mock2 = await PickACard.new(this.proxyAddress2);
```
Comment (facuspagnuolo) [inline kernel/test/KernelProxy.test.js]: what about using `anotherMock` here?

File: kernel/test/KernelProxy.test.js [hunk 2]
```diff
@@ -69,4 +69,14 @@ contract('zeppelin_os', ([_, zeppelin, developer, someone, anotherone]) => {
     await this.mock2.pick(5, { from: anotherone }).should.be.fulfilled;
   });
 
+  it('storage takes place in KernelProxy', async function () {
+    //fetch the owner of token 5, in `mapping (uint256 => address) internal tokenOwner;`
+    var ind = '0000000000000000000000000000000000000000000000000000000000000002' // tokenOwner position in storage
+    var key =  '0000000000000000000000000000000000000000000000000000000000000007' // tokenId
+    var newkey =  web3.sha3(key + ind, {"encoding":"hex"})
```
Comment (facuspagnuolo) [inline kernel/test/KernelProxy.test.js]: can we use `const` here?

File: kernel/test/KernelProxy.test.js [hunk 3]
```diff
@@ -69,4 +69,14 @@ contract('zeppelin_os', ([_, zeppelin, developer, someone, anotherone]) => {
...
+    await this.mock2.pick(7, { from: someone });
+    var storage = await web3.eth.getStorageAt(this.proxyAddress2, newkey);
```
Comment (facuspagnuolo) [inline kernel/test/KernelProxy.test.js]: same here

**Comment by facuspagnuolo** [comment]
@fiiiu we will need to rebase this branch from master

File: kernel/test/KernelProxy.test.js (hunk 1 above)
Comment (fiiiu) [reply kernel/test/KernelProxy.test.js]: I changed `this.proxyAddress2` to `this.anotherProxyAddress` as well, despite the ambiguity as to whether it is a new proxy or a new address of the same proxy, which is in any case also present in `proxyAddress2`.

**Pull request merged**